apps/sensenet/src/localization/default.ts

```diff
@@ -118,7 +118,7 @@ const values = {
   drawer: {
     titles: {
       Content: 'Content',
-      ContentTypes: 'Content Types',
+      ContentTypes: 'Content Explorer',
```

Comment:
Not this one, the Content Types title was OK. We meant to change the CustomContent menu item below to Content Explorer.